conductor-client/src/main/kotlin/com/openlattice/organizations/roles/HazelcastPrincipalService.kt

```diff
@@ -72,13 +75,12 @@ class HazelcastPrincipalService(
         }
 
         private fun hasSecurablePrincipal(principalAclKey: AclKey): Predicate<AclKey, AclKeySet> {
-            return Predicates.equal("this.index[any]", principalAclKey.index)
+            return Predicates.equal("this.${PrincipalTreesMapstore.INDEX}", principalAclKey.index)
```

Comment:
I'm pretty sure the `this.` isn't necessary (and if it is, it should probably be a part of `PrincipalTreesMapstore.INDEX`)